frontend/src/components/Content.tsx

```diff
@@ -44,6 +44,7 @@ export default function Content() {
           alignItems: 'center',
           gap: 1,
         }}>
+          { /*
```

Comment:
Are we not keeping the connection modal , for user to connect ,disconnect and select database with NEO4j ?

Reply:
until we're making it optional and also allows sending that information to the backend it's better to disable it

Reply:
sure will handle that